### PR TITLE
Fixed footer alignment and unnecessary warnings

### LINKED ISSUE #24: Inappropriate footer alignment
**Description**
Footer is not aligned to the center of the viewport.

**Screenshot**
footer

### PR DESCRIPTION
**Pull request by Samridhi-98**
**Description**
Fixed footer alignment and unnecessary warnings

Fixes #24 

**Changes**
- [x] Updated `PersistentDrawerLeft.js`
- [x] Modified `PersistentDrawerLeft.css`
- [x] Modified `BreadCrum.css`
 
**Screenshot**

image_2021_05_03T19_21_12_263Z

image_2021_05_03T19_20_16_633Z

### CODE REVIEW

**Comment by Samridhi-98** [comment]
@ArunTeltia  @LordShenron  Please review.

**Comment by LordShenron** [comment]
We will review it soon.

**Comment by LordShenron** [comment]
can we go ahead with it @ArunTeltia

File: src/components/Guides/BuildSakura/BuildSakura.js [hunk 1]
```diff
@@ -1,98 +1,98 @@
...
-            can setup the build environment using
-            <a href="https://github.com/akhilnarang/scripts" target="_blank">
-              {" "}
+            <hr />
+            <h2>HOW TO SETUP A BUILD ENVIRONMENT</h2>
+            <p>
+              To Build Project Sakura you can use any 64 Bit Linux Distro, though
+              we recommend using Ubuntu 18.04, LinuxMint, MXLinux or Manjaro. You
+              can setup the build environment using
+            <a href="https://github.com/akhilnarang/scripts" target="_blank" rel="noreferrer">
```
Comment (ArunTeltia) [inline src/components/Guides/BuildSakura/BuildSakura.js:33]: can you tell me what rel="noreferrer" doing i am intrested to know

BTW the PR looks good to me
Just one advice can you not include the indentation changes in the next PR, its hard to review @Samridhi-98 

Nevertheless solid work

**Comment by ArunTeltia** [comment]
It can be merged @LordShenron

File: src/components/Guides/BuildSakura/BuildSakura.js (hunk 1 above)
Comment (Samridhi-98) [reply src/components/Guides/BuildSakura/BuildSakura.js:33]: ` rel="noreferrer"` is basically used to protect the user from hijacking while browsing your site as links to cross-origin destinations are unsafe.
That indentation occurs due to my vscode extension, I will disable it while working on sakura if you say so.

**Pull request merged**